ci/install.sh

```diff
@@ -99,35 +102,127 @@ if [ ! -e "${GPG21_INSTALL}/bin/gpg" ]; then
-    ./configure --prefix="${GPG21_INSTALL}" --with-libgpg-error-prefix="${GPG21_INSTALL}"
+    ./configure --prefix="${GPG_INSTALL}" --with-libgpg-error-prefix="${GPG_INSTALL}"
     ${MAKE} -j${CORES} install
     cd ..
   done
 
-  wget -c https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.1.0.tar.bz2
-  wget -c https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.1.0.tar.bz2.sig
-  gpg --verify pinentry-1.1.0.tar.bz2.sig
-  tar -xjf pinentry-1.1.0.tar.bz2
-  cd pinentry-1.1.0
-  ./configure --prefix="${GPG21_INSTALL}" \
-    --with-libgpg-error-prefix="${GPG21_INSTALL}" \
-    --with-libassuan-prefix="${GPG21_INSTALL}" \
+  wget -c https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-${PINENTRY_VERSION}.tar.bz2
+  wget -c https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-${PINENTRY_VERSION}.tar.bz2.sig
+  gpg --verify pinentry-${PINENTRY_VERSION}.tar.bz2.sig
+  tar -xjf pinentry-${PINENTRY_VERSION}.tar.bz2
+  cd pinentry-${PINENTRY_VERSION}
+  ./configure --prefix="${GPG_INSTALL}" \
+    --with-libgpg-error-prefix="${GPG_INSTALL}" \
+    --with-libassuan-prefix="${GPG_INSTALL}" \
     --enable-pinentry-curses \
     --disable-pinentry-qt4
   ${MAKE} -j${CORES} install
   cd ..
 
-  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.4.tar.bz2
-  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.4.tar.bz2.sig
-  gpg --verify gnupg-2.2.4.tar.bz2.sig
-  tar -xjf gnupg-2.2.4.tar.bz2
-  cd gnupg-2.2.4
-  ./configure --prefix="${GPG21_INSTALL}" \
-    --with-libgpg-error-prefix="${GPG21_INSTALL}" \
-    --with-libgcrypt-prefix="${GPG21_INSTALL}" \
-    --with-libassuan-prefix="${GPG21_INSTALL}" \
-    --with-ksba-prefix="${GPG21_INSTALL}" \
-    --with-npth-prefix="${GPG21_INSTALL}"
+  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-${GNUPG_VERSION}.tar.bz2
+  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-${GNUPG_VERSION}.tar.bz2.sig
+  gpg --verify gnupg-${GNUPG_VERSION}.tar.bz2.sig
+  tar -xjf gnupg-${GNUPG_VERSION}.tar.bz2
+  cd gnupg-${GNUPG_VERSION}
+  ./configure --prefix="${GPG_INSTALL}" \
+    --with-libgpg-error-prefix="${GPG_INSTALL}" \
+    --with-libgcrypt-prefix="${GPG_INSTALL}" \
+    --with-libassuan-prefix="${GPG_INSTALL}" \
+    --with-ksba-prefix="${GPG_INSTALL}" \
+    --with-npth-prefix="${GPG_INSTALL}"
   ${MAKE} -j${CORES} install
 
+}
+
+build_gpg_beta() {
+  mkdir gettext && cd gettext
+  gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D7E69871
+  wget -c https://ftp.gnu.org/pub/gnu/gettext/gettext-latest.tar.xz
+  wget -c https://ftp.gnu.org/pub/gnu/gettext/gettext-latest.tar.xz.sig
+  gpg --verify gettext-latest.tar.xz.sig
+  tar -xJf gettext-latest.tar.xz --strip 1
+  ./configure --prefix="${GPG_INSTALL}"
+  ${MAKE} -j${CORES} install
+  cd ..
+  export GETTEXT_PREFIX="${GPG_INSTALL}/bin/"
+
+  # workaround https://github.com/travis-ci/travis-ci/issues/8613
+  # alternatively, forcing the libgpg-error build to use gcc should work
+  export LD_LIBRARY_PATH="/usr/local/clang-5.0.0/lib"
```

Comment:
I realize this is also not ideal and may require an update at soem point. Hopefully travis will fix the issue here. (I also tried this with globbing, rather than hardcoding the version, but it ended up set to the literal value, maybe travis is doing something weird there for security.)